jekyll/_cci2/building-docker-images.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: classic-docs
-title: "Running Docker Commands"
-short-title: "Running Docker Commands"
+title: "Building Docker Images"
```

Comment:
Happy to switch this back depending on what others think. Building Docker Images just makes more sense to me as a title.

Reply:
I believe there are other docker commands a user might be using, though (although most prevalently ... `build`-ing...) I'm not super versed on Docker stuff, though.

Reply:
Oh yeah! Good point 👍